README.md

```diff
@@ -165,8 +165,8 @@ def generate_sitemap():
 And you will need to configure xml files static responses, i.e. in nginx:
 
 ```
-location ~* /sitemaps/(?<fn>sitemap(-(article|video)).xml {
-    try_files /media/sitemaps/$fn$arg_p.xml @backend;
+location ~^/sitemaps/(?<fn>sitemap(-(article|video))?.xml) {
+    alias /media/sitemaps/$fn;
```

Comment:
1. try_files was here to pass cache-miss requests to backend (at least for testing purposes)
2. note that you wiped page number argument from example, so all sitemap pages will be the same
3. also there is no `~^` operator in [nginx](https://nginx.org/en/docs/http/ngx_http_core_module.html), did you mean `~ ^`?

```
    location ~* /sitemaps/(?<fn>sitemap(-(article|category|tag))?).xml {
        # sitemap-article.xml?p=2 searching at sitemap-article2.xml
        # sitemap-article.xml searching at sitemap-article.xml
        try_files /media/sitemaps/$fn$arg_p.xml @backend;
    }
```

Code above works in production, at least as I know.